src/pudl/convert/epacems_to_parquet.py

```diff
@@ -116,16 +106,10 @@ def main():
         clobber=args.clobber,
     )
 
-    # Messy settings arguments used in our main ETL which we should clean up
-    epacems_etl_settings = pudl.settings.EpaCemsBaseSettings(
```

Comment:
As far as I could tell this `EpaCemsBaseSettings` model no longer existed. Not sure why it wasn't throwing an error.